routers/register.py

```diff
@@ -99,7 +99,13 @@ async def qr_validation(authenticated_user: AuthenticatedUser) -> dict:
         - **ValueError**:
             If email is not valid
     """
-    return validate_qr_registration(user.email, user.qr_value)
+    is_validate = validate_qr({"email": authenticated_user.email}, authenticated_user.qr_value)
+
+    if is_validate:
+        send_email(authenticated_user.email)
+        # TODO: Check Data Response
+        return "Check your email to finish the registration process"
```

Comment:
This was reported as a code smell: 

Return a value of type "dict" instead of "str" or update function "qr_validation" type hint